python/lsst/galaxy_shear/processShearTest.py

```diff
@@ -117,22 +147,63 @@ def run(self, dataRef):
             bbox = lsst.afw.geom.Box2I(lsst.afw.geom.Point2I(x,y), self.dims)
             exp = lsst.afw.image.ExposureF(exposure, bbox)
             exp.setPsf(psf)
+            CD = numpy.array([[5.55E-5, 0.0], [0.0, 5.55E-5]])
+            crpix = lsst.afw.geom.Point2D(0.0,0.0)
+            crval = lsst.afw.geom.Point2D(0.0,0.0)
+            exp.setWcs(lsst.afw.image.Wcs(crval, crpix, CD))
+            calib = lsst.afw.image.Calib()
+            calib.setFluxMag0((3531360874589.57, 21671681149.139))
+            exp.setCalib(calib)
             #  Now do the measurements, calling the measure algorithms to increase speed
+            sigma = 0 
             try:
-                if self.centroidPlugin:
-                    plugin = self.centroidPlugin
-                    plugin.measure(source, exp)
-                if self.measPlugin:
-                    plugin = self.measPlugin
-                    plugin.measure(source, exp)
+                for plugin in self.measurement.plugins.keys():
+                    self.measurement.plugins[plugin].measure(source, exp)
+                if self.e1Key != None:
+                    e1 = source.get(self.e1Key)
+                    e2 = source.get(self.e2Key)
+                if self.sigmaKey != None:
+                    sigma = source.get(self.sigmaKey)
+
+                if self.xxKey != None:
+                    xx = source.get(self.xxKey)
+                    yy = source.get(self.yyKey)
+                    xy = source.get(self.xyKey)
+                    e1 = (xx - yy) / (xx + yy)
+                    e2 = 2 * xy / (xx + yy)
+
+                e = math.sqrt(e1 * e1 + e2 * e2)
```

Comment:
There should be spaces between + and - but not between *, /, and %.  This is throughout.